docs/USAGE.rst

```diff
@@ -356,29 +356,29 @@ These business rules will be automatically applied at the time of data import to
 
 The following rules apply to all file formats:
 
-* Tiff DateTime (if present), set to 01:01:(year and time)
-* Copyright: removed
-* HostComputer: removed
-* tiff.Software: modified by adding custom tags with data from the DeID Upload file
+* Tiff ``DateTime`` (if present), set to 01:01:(year and time)
```

Comment:
I think that the entire field is "Tiff DateTime", so the double ticks should include the entire phrase.

Reply:
@manthey 
Would it be more appropriate to change it to `tiff.DateTime`? I'm looking at `process.py` here, particularly the last line:
```
    for key in {'DateTime'}:
        tag = tifftools.Tag[key].value
        if tag in ifds[0]['tags']:
            value = ifds[0]['tags'][tag]['data']
            if len(value) >= 10:
                value = value[:5] + '01:01' + value[10:]
            else:
                value = None
            redactList['metadata']['internal;openslide;tiff.%s' % key] = {'value': value}
```

It actually looks like Copyright and HostName also follow this format. Should the docs be updated to reflect that as well?

Reply:
Yes, you are right.  The docs probably should all be `tiff.<key>`.